### PR TITLE
match invite members button space with other lhs spacing

### PR DESCRIPTION
**Pull request by neallred**
#### Summary
Fix whitespace of left hand side invite members button to match other left hand side items.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-38466

#### Related Pull Requests
none

#### Screenshots

before:
lhs-button-after

after:
lhs-button-after

#### Release Note

```release-note
Fix whitespace of left hand side invite members button to match other left hand side items.
```

### CODE REVIEW

**Review by esethna — APPROVED** [review]
Thanks!

**Review by stevemudie — APPROVED** [review]
Looks good @neallred!

- Created a new category and moved it around
- Created a GM as well and compared it to channels above etc.
- Everything looked good and spacing appeared consistent

File: components/sidebar/invite_members_button.tsx [hunk 1]
```diff
@@ -30,7 +34,7 @@ const InviteMembersButton: React.FC = (): JSX.Element => {
             <ToggleModalButtonRedux
                 accessibilityLabel={intl.formatMessage({id: 'sidebar_left.inviteUsers', defaultMessage: 'Invite Users'})}
                 id='introTextInvite'
-                className='intro-links color--link cursor--pointer'
+                className={`intro-links color--link cursor--pointer${props.className ? ` ${props.className}` : ''}`}
```
Comment (pablovelezvidal) [inline components/sidebar/invite_members_button.tsx:37]: Not sure if the surrounded backticks around ${props.className} is actually needed?

**Review by pablovelezvidal — APPROVED** [review]
Thanks @neallred ! Just one minor question

File: components/sidebar/invite_members_button.tsx (hunk 1 above)
Comment (neallred) [reply components/sidebar/invite_members_button.tsx:37]: Depends. I didn't want to have the extra space in the case of no passed in class name, and I didn't want to have to use `classnames` or some lib to trim the extra space, so this is the way I thought of to do that. Open to alternate forms of achieving the same thing (including using `classnames` repo).

**Review by AGMETEOR — APPROVED** [review]

**Pull request merged**